_pages/author-tiril.html

```diff
@@ -0,0 +1,42 @@
+---
+title: "Tiril"
+layout: default
+permalink: "/author-tiril.html"
+---
+
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-md-8">
+      <div class="row align-items-center mb-5">
+        <div class="col-md-9">
+          <h2 class="font-weight-bold">
+            {{page.title}}
+            <span class="small btn btn-outline-success btn-sm btn-round"
+              ><a href="{{ site.authors.tiril.twitter }}">Follow</a></span
```

Comment:
Use `linkedin` property from profile entry:

```suggestion
              ><a href="{{ site.authors.tiril.linkedin }}">Follow</a></span
```

Reply:
I did that but the icon was the twitter bird? So I reverted it again

Reply:
Ah, it's the hardcoded icon handling here, which is the problem: https://github.com/tech-women/tech-women.github.io/blob/3c3a5f6934ca5c2c3eb3cee30a56f09f3d3be0ae/_pages/authors-list.html#L34